src/main/java/com/thealgorithms/strings/LetterCombinationsOfPhoneNumber.java

```diff
@@ -5,49 +5,55 @@
 import java.util.List;
 
 public final class LetterCombinationsOfPhoneNumber {
+
+    // Mapping of numbers to corresponding letters on a phone keypad
+    private static final String[] KEYPAD = new String[] {" ", "", "abc", "def", "ghi", "jkl", "mno", "pqrs", "tuv", "wxyz"};
```

Comment:
```suggestion
    private static final String[] KEYPAD = new String[] {"", "", "abc", "def", "ghi", "jkl", "mno", "pqrs", "tuv", "wxyz"};
```

I think this space was introduced by accident in 06d49ced305423dba456fcb710a74506b2169b57.